java/tools/src/test/org/apache/orc/tools/convert/TestJsonReader.java

```diff
@@ -72,4 +75,25 @@ public void testTimestampOffByOne() throws Exception {
         assertEquals("1969-12-31 23:59:58.9999", cv.asScratchTimestamp(5).toString());
     }
 
+    @Test
+    public void testDateTypeSupport() throws IOException {
```

Comment:
Can we add a case for null rows, repeating rows etc.?

Reply:
added